src/test/java/marquez/api/mappers/DatasetResponseMapperTest.java

```diff
@@ -0,0 +1,47 @@
+package marquez.api.mappers;
+
+import static marquez.common.models.Description.NO_DESCRIPTION;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.time.Instant;
+import java.util.Optional;
+import marquez.api.models.DatasetResponse;
+import marquez.common.models.DatasetUrn;
+import marquez.common.models.Description;
+import marquez.service.models.Dataset;
+import org.junit.Test;
+
+public class DatasetResponseMapperTest {
+  private static final DatasetUrn DATASET_URN = DatasetUrn.of("urn:a:b.c");
+  private static final Instant CREATED_AT = Instant.now();
+
+  @Test
+  public void testMapDatasetResponse() {
+    final Description nonEmptyDescription = Description.of("test description");
+    final Optional<String> nonEmptyDescriptionString = Optional.of(nonEmptyDescription.getValue());
+    final Dataset dataset = new Dataset(DATASET_URN, CREATED_AT, nonEmptyDescription);
+    final DatasetResponse datasetResponse = DatasetResponseMapper.map(dataset);
+    assertNotNull(datasetResponse);
+    assertEquals(DATASET_URN.getValue(), datasetResponse.getUrn());
+    assertEquals(CREATED_AT.toString(), datasetResponse.getCreatedAt());
+    assertEquals(nonEmptyDescriptionString, datasetResponse.getDescription());
+  }
+
+  @Test
+  public void testMapDatasetResponseNoDescription() {
+    final Optional<String> noDescriptionString = Optional.of(NO_DESCRIPTION.getValue());
+    final Dataset dataset = new Dataset(DATASET_URN, CREATED_AT, NO_DESCRIPTION);
+    final DatasetResponse datasetResponse = DatasetResponseMapper.map(dataset);
+    assertNotNull(datasetResponse);
+    assertEquals(DATASET_URN.getValue(), datasetResponse.getUrn());
+    assertEquals(CREATED_AT.toString(), datasetResponse.getCreatedAt());
+    assertEquals(noDescriptionString, datasetResponse.getDescription());
```

Comment:
Maybe also worth asserting that `Optional.of(NO_DESCRIPTION.getValue());` is equal to the `NO_VALUE` string and that `datasetResponse.getDescription()` is also equal to that.